### DISCUSSION TITLE
fattureincloud sdk ListIssuedDocuments Query format

### DISCUSSION THREAD

**Discussion by Fulvioavatar** [fattureincloud/api#292] [Q&A] [answered]
I've put this code in visual studio c#: 
var apiInstance = new IssuedDocumentsApi(config);
            var companyId = ******;  // int | The ID of the company.
            var type = "invoice";  // string | The type of the issued document.
            var fields = "Id";  // string | List of comma-separated fields. (optional) 
            var fieldset = "basic";  // string | Name of the fieldset. (optional) 
            var sort = "-id";  // string | List of comma-separated fields for result sorting (minus for desc sorting). (optional) 
            var page = 1;  // int? | The page to retrieve. (optional)  (default to 1)
            var perPage = 5;  // int? | The size of the page. (optional)  (default to 5)
            var q = "";  // string | Query for filtering the results. (optional) 
            var inclusive = 0;  // int? | (Only for type = delivery_notes) Include invoices delivery notes. (optional) 
            ListIssuedDocumentsResponse Fatture = apiInstance.ListIssuedDocuments(companyId, type, fields, fieldset, sort, page, perPage, q, inclusive);
            Console.WriteLine("id fattura: " + Fatture.Data[0].Id + ", anno fattura: " + Fatture.Data[0].Date + ", numero fattura: " + Fatture.Data[0].Number);

I want to filter with query usage, but i'm not able to find example code or corret syntax to implement to, someone can help me?
I'm italian so you can write in italian, very thanks!

**Comment by emajo** [comment, marked as answer]
Hi @Fulvioavatar, you can find the information you need about the filters in the documentation [here](https://developers.fattureincloud.it/docs/basics/additional-capabilities/filter-results).

Thanks,
Emanuele.

**Reply by Fulvioavatar** [reply]
Very thanks!!
